docker/docker-compose-elk.yml

```diff
@@ -66,21 +66,38 @@ services:
     build: "../on-http"
     depends_on:
       - files
+      - logstash
       - mongo
       - rabbitmq
+      - statsd
+      - tasks
```

Comment:
on-http truely depends on tasks, but the denpendency has been defined in Dockerfile of on-http.

If indicate dependency here then the http will wait for tasks running container, and IMO this is not expected here.

Suggestion: remove tasks.